README.md

```diff
@@ -86,6 +86,7 @@ The following environment variables tweak DXVK-NVAPI's runtime behavior:
   - `TU100` (Turing)
   - `GA100` (Ampere)
   - `AD100` (Ada)
+  - `GB100` (Blackwell)
```

Comment:
This is incorrect, we are actually looking for `GB200`, not `GB100`.

Reply:
Thanks, rebased again on top of the original PR which includes the correct line.